tests/csm/rest/test_query_deployment.py

```diff
@@ -309,6 +280,10 @@ def test_45670(self):
         test_case_name = cortxlogging.get_frame()
         self.log.info("##### Test started -  %s #####", test_case_name)
         self.log.info("Step 1: Verify GET system topology ")
-        result, err_msg = self.csm_obj.verify_system_topology(expected_response=HTTPStatus.OK)
+        deploy_start_time = str(self.deploy_start_time).split('.', maxsplit=1)[0]
```

Comment:
Why do we need this time?
Time should be compared in int

Reply:
time.time gives float and time from response is str epoch format, will convert both to int and compare

Reply:
u dont need float to be converted to int for comparison. U can convert str to int or float as applicable

Reply:
done